modules/402-ingress-nginx/monitoring/prometheus-rules/ingress-nginx.yaml

```
@@ -248,6 +248,11 @@
         To re-enable validation, remove the annotation `network.deckhouse.io/ingress-nginx-validation-suspended`
         from the `ingressnginxcontroller` resource.
 
+        To find which `ingressnginxcontroller` resources have the annotation, use the following command:
+        ```shell
+          d8 k get ingressnginxcontrollers.deckhouse.io -o json | jq -r '.items[] | select(.metadata.annotations."network.deckhouse.io/ingress-nginx-validation-suspended" != null) | "\(.metadata.name)"'
+        ```
+
```

```suggestion
        To find which `ingressnginxcontroller` resources have the annotation, use the following command:
        ```shell
        d8 k get ingressnginxcontrollers.deckhouse.io -o json | jq -r '.items[] | select(.metadata.annotations."network.deckhouse.io/ingress-nginx-validation-suspended" != null) | "\(.metadata.name)"'
        ```

```